src/db_options.rs

```diff
@@ -4886,6 +4888,45 @@ impl Drop for DBPath {
     }
 }
 
+/// Options for importing column families. See
+/// [DB::create_column_family_with_import](crate::DB::create_column_family_with_import).
+pub struct ImportColumnFamilyOptions {
+    pub(crate) inner: *mut ffi::rocksdb_import_column_family_options_t,
+}
+
+impl ImportColumnFamilyOptions {
+    pub fn new() -> Self {
+        let inner = unsafe { ffi::rocksdb_import_column_family_options_create() };
+        ImportColumnFamilyOptions { inner }
+    }
+
+    /// Determines whether to move the provided set of files on import. The default
+    /// behavior is to copy the external files on import. Setting `move_files` to `true`
+    /// will move the files instead of copying them. See
+    /// [DB::create_column_family_with_import](crate::DB::create_column_family_with_import)
+    /// for more information.
+    pub fn set_move_files(&mut self, move_files: bool) {
+        unsafe {
+            ffi::rocksdb_import_column_family_options_set_move_files(
+                self.inner,
+                c_uchar::from(move_files),
+            );
+        }
+    }
+}
+
+impl Default for ImportColumnFamilyOptions {
+    fn default() -> Self {
+        Self::new()
```

Comment:
would be good to do:
```
        let inner = unsafe { ffi::rocksdb_export_import_files_metadata_create() };
        assert!(
            !inner.is_null(),
            "Could not create rocksdb_export_import_files_metadata_t"
        );
        Self { inner }
```

to match the pattern that other code uses. see `impl Default for BlockBasedOptions` for example.